CMakeLists.txt

```diff
@@ -459,15 +459,19 @@ set(IO_SOURCE
 set(IO_ADIOS1_SEQUENTIAL_SOURCE
         src/Error.cpp
         src/auxiliary/Filesystem.cpp
+        src/auxiliary/JSON.cpp
         src/ChunkInfo.cpp
         src/IO/ADIOS/CommonADIOS1IOHandler.cpp
-        src/IO/ADIOS/ADIOS1IOHandler.cpp)
+        src/IO/ADIOS/ADIOS1IOHandler.cpp
+        src/IO/IOTask.cpp)
```

Comment:
Needed because we use `openPMD::internal::operationAsString` in `CommonADIOS1IOHandler.cpp`